packages/config/lib/options.ts

```diff
@@ -502,6 +502,10 @@ export const breakingOptions: Array<BreakingOption> = [
     name: 'experimentalRunEvents',
     errorKey: 'EXPERIMENTAL_RUN_EVENTS_REMOVED',
     isWarning: true,
+  }, {
+    name: 'experimentalSessionSupport',
+    errorKey: 'EXPERIMENTAL_SESSION_SUPPORT_REMOVED',
+    isWarning: true,
```

Comment:
Does this mean tests previously using sessions will continue to be able to use cy.session and pass? If these changes do not allow the cy.session command to be enabled when this config key exists, I'd expect this to throw an error to give users a clear path forward vs their tests running and failing when cy.session is used.

Reply:
Upon release, the `experimentalSessionSupport` flag will be removed, and cause a warning if used. The behaviour then will be that any tests using `cy.session` will fail with a warning telling users to enable the new flag `experimentalSessionAndOrigin`.